game/game.py

```diff
@@ -386,84 +386,111 @@ def set_bullseye(self) -> None:
         self.blue_bullseye = Bullseye(enemy_cp.position)
         self.red_bullseye = Bullseye(player_cp.position)
 
-    def initialize_turn(self) -> None:
+    def initialize_turn(
+        self,
+        for_coalition: List[str] = ["Blue", "Red"],
+        is_reinit: bool = False,
+        clear_existing: bool = True,
```

Comment:
This is never called with anything but the default. Better to remove the untested False option entirely.

Reply:
this came due to a merge from my other changeset where i am using this function for filling unassigned aircraft with the ai mission generation - was not expected to be here at all yet